### PR TITLE
Shiny app

### PR DESCRIPTION
**Pull request by yte0**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -25,6 +25,11 @@ Suggests:
     covr
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.0.2
+Depends:
+    R (>= 2.10.0)
```
Comment (arnfinn) [inline DESCRIPTION]: This has been changed in PR #28. Thus, has to remove these two lines after update.

File: NAMESPACE [hunk 2]
```diff
@@ -2,7 +2,20 @@
...
+export(filter_data)
+export(ki_table)
+export(load_data)
+export(mod_kvalitetsoversikt_server)
+export(mod_kvalitetsoversikt_ui)
 export(run_app)
+export(table_row_constructor)
 importFrom(magrittr,"%>%")
+importFrom(rlang,.data)
+importFrom(shiny,NS)
```
Comment (arnfinn) [inline NAMESPACE]: Is this needed anymore?

Comment (arnfinn) [inline NAMESPACE]: Is `importFrom(shiny,NS)` still needed?

Comment (areedv) [reply NAMESPACE]: NAMESPACE is generated from function docs and any changes should be made there...

File: R/app_server.R [hunk 3]
```diff
@@ -8,4 +8,6 @@
 #' @export
 app_server <- function(input, output, session) {
   # List the first level callModules here
+  shiny::callModule(qmongr::mod_kvalitetsoversikt_server,
```
Comment (areedv) [inline R/app_server.R]: stick with English as far as possible

File: R/mod_kvalitetsoversikt.R [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+#' @param id shiny id
+#' @param input internal
+#' @param output internal
+#' @param session internal
+#'
+#' @rdname mod_kvalitetsoversikt
+#'
+#' @keywords internal
+#' @export
+#' @importFrom shiny NS tagList
```
Comment (arnfinn) [inline R/mod_kvalitetsoversikt.R]: rm @importFrom shiny NS tagList?

**Pull request merged**